packages/frontend/src/components/atoms/button.atom.tsx

```
@@ -0,0 +1,53 @@
+
+  const sizes = {
+    sm: 'text-sm px-3 py-1.5 rounded-lg',
+    md: 'text-base px-4 py-2 rounded-xl',
+    lg: 'text-base px-6 py-4 rounded-xl',
+  };
+
+  return (
+    <HeadlessButton className={cn(baseStyles, variants[variant], sizes[size], fullWidth && 'w-full', className)} disabled={disabled || loading} ref={ref} {...props}>
+      {loading ? (
+        <>
+          <svg className="-ml-1 mr-3 h-5 w-5 animate-spin text-current" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)